src/IssueCreator/IssueManager.cs

```diff
@@ -184,19 +201,31 @@ internal async Task<(bool, string)> TryCreateNewIssueAsync(IssueToCreate issueTo
                 // assign the epic, if one is selected
                 if (issueToCreate.Epic != null)
                 {
-                    await AddIssueToEpicAsync(issueToCreate.Epic.Repo.Id, issueToCreate.Epic.Issue.Number, repoFromGH.Id, createdIssue.Number);
+                    bool result = await AddIssueToEpicAsync(issueToCreate.Epic.Repo.Id, issueToCreate.Epic.Issue.Number, repoFromGH.Id, createdIssue.Number);
+                    if (!result)
+                    {
+                        MessageBox.Show("Cannot associate issue to epic");
```

Comment:
Maybe surface any exception message in the error display. More information would make it easier to diagnose a problem.

Reply:
Hmm... in that case, then maybe the right thing to do is have the leaf methods throw

Reply:
But that would then fail the whole operation, right? At least for ZenHub operations, at least the GH issue was created, but still open the browser. It makes sense to at least inform the user that ZH failed so they know to manually fix up the GH-created issue. Basically, the current flow seems right but I just didn't know the ZH portion failed, or I'd have fixed it up.

Still curious why it failed for at least 2 separate events of multiple bugs each. Hence why giving me some server response in the message box so I can maybe have a chance at fixing the underlying issue (maybe I need to generate a new PAT, even though I'm able to query epics).

Reply:
I don't think this is about the PAT for ZenHub. I also saw this failure on my machine, and I didn't have an indication of what went wrong... Hopefully additional logging will help here.

Reply:
Given it's an internal tool and UI isn't as important, a message box with even the full `Exception.ToString()` (but that didn't otherwise fail - getting as much done as possible/sensible) would be easy to diagnose. I've had it happen at least twice with a week or two in between, so it's likely not a transient issue (though, in the middle of a workday, could be server strain or something).